_data/toc/release-notes.yml

```diff
@@ -266,13 +266,13 @@ pages:
       children:
 
         - label: Magento Open Source third-party licenses
```

Comment:
Since this topic now contains magento packages in addition to third-party packages, the TOC labels don't make sense. Should it be "Magento <edition> packages" or "Magento <edition> licenses" or something else?

Reply:
I think "Packages" will be good.